docs/rules/README.md

```diff
@@ -190,6 +190,7 @@ These rules are purely matters of style and are quite subjective.
 * [no-nested-ternary](no-nested-ternary.md): disallow nested ternary expressions
 * [no-new-object](no-new-object.md): disallow `Object` constructors
 * [no-plusplus](no-plusplus.md): disallow the unary operators `++` and `--`
+* [no-restricted-properties](no-restricted-properties.md) - disallow use of specified object properties
```

Comment:
disallow the use of specified object properties

After recent editing, most (but not quite all) occurrences of use are preceded by the.
Extra word :( but helps make clear it is a noun not a verb, especially if English is not first language of reader :)